packages/phasor/src/utils/math-utils.ts

```diff
@@ -499,3 +499,21 @@ export function isOverlap(
     )
   );
 }
+
+export function calcPointToPointWithAngle(
```

Comment:
there already has `rotatePoints` method in math-utils. you can use it to avoid duplication

Reply:
Ok. I will update this PR later